camelot/model/fixture.py

```diff
@@ -226,9 +226,8 @@ class FixtureVersion( Entity ):
     __tablename__ = 'fixture_version'
     
     fixture_version = Column( Integer, index = True, nullable=False, default=0 )
-    fixture_class = Column( Unicode( 255 ), index = True, nullable=True,
-                            primary_key=True )
-    
+    fixture_class = Column( Unicode( 255 ), index = True, nullable=False, primary_key=True)
```

Comment:
In 1.3 zorgde het primary_key=True argument er voor dat de kolom automatisch non-nullable was, in 1.4. niet meer. In de migratie is de kolom altijd non-nullable geweest.

Reply:
die hele fixture bazaar mag toch weg ...